config/routes.rb

```diff
@@ -50,6 +50,10 @@
 
   get "/documentation" => "documentation#index"
 
+  get "/publisher-information/how-to" => "publisher_information#how_to", as: :how_to
+  get "/publisher-information/beta-capabilities" => "publisher_information#beta_capabilities", as: :beta_capabilities
+  get "/publisher-information" => "publisher_information#index", as: :publisher_updates
```

Comment:
Rather than index might be good to have a publisher_updates action rather than index?

Also for all 3 of these might be good to give them dedicated routes like `/how-to-use` `/beta-capabilities` `/publisher-updates`